owasp-top10-2017-apps/a5/tictactoe/README.md

```diff
@@ -0,0 +1,271 @@
+
+# Tic-Tac-Toe
+
+  
+
+<p  align="center">
+
+<img  src="images/a5-banner.png"/>
+
+</p>
+
+  
+
+Tic-Tac-Toe is a simple Nodejs web application that contains an example of a Broken Access Control vulnerability and its main goal is to describe how a malicious user could exploit it.
+
+  
+
+## Index
+
+  
+
+-  [Definition](#what-is-broken-access-control)
+
+-  [Setup](#setup)
+
+-  [Attack narrative](#attack-narrative)
+
+-  [Objectives](#secure-this-app)
+
+-  [Solutions](#pr-solutions)
+
+-  [Contributing](#contributing)
+
+  
+
+## What is Broken Access Control?
+
+  
+
+Restrictions on what authenticated users are allowed to do are often not properly enforced. Attackers can exploit these flaws to access unauthorized functionality and/or data, such as access to other users' accounts, view sensitive files, modify other users’ data, change access rights, etc.
+
+  
+
+The main goal of this app is to discuss how **Broken Access Control** vulnerabilities can be exploited and to encourage developers to send secDevLabs Pull Requests on how they would mitigate these flaws.
+
+  
+
+## Setup
+
+  
+
+To start this intentionally **insecure application**, you will need [Docker][Docker Install] and [Docker Compose][Docker Compose Install]. After forking [secDevLabs](https://github.com/globocom/secDevLabs), you must type the following commands to start:
+
+  
+
+```sh
+
+cd secDevLabs/owasp-top10-2017-apps/a5/tictactoe
+
+```
+
+  
+
+```sh
+
+make install
+
+```
+
+  
+
+Then simply visit [localhost:10005][App] ! 😆
+
+  
+
+## Get to know the app 💵
```

Comment:
I think 👾, 🕹, or 🎮 would be better than 💵

Reply:
I choose the second one :)